introduction.md

```diff
@@ -4,3 +4,25 @@ I just finished the db path
 i wanna finish the git path :-/
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?